src/modules/vehicle/engine.h

```diff
@@ -40,6 +41,7 @@ namespace MM2
         Matrix34 *EngineVisualMatrixPtr;
         vehCarSim *VehCarSimPtr;
         vehTransmission *VehTransmissionPtr;
+        vehDriveTrain *VehDriveTrainPtr;
```

Comment:
Where does this come from? vehEngine appears to have a size of 0x84 (you can see by how it's initialized)